public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionCompleteProgressBar.js

```diff
@@ -71,6 +71,11 @@ function ModalMissionCompleteProgressBar (uiModalMissionComplete) {
      * @private
      */
     this.update = function (missionDistanceRate, userAuditedDistanceRate, otherAuditedDistanceRate) {
+        // rounding rates to .1 accuracy to avoid floating value arithmetic error
+        missionDistanceRate = missionDistanceRate.toFixed(1);
```

Comment:
You should avoid overwriting the values of input parameters. Instead, you should create new variables and (e.g., `roundedMissionDistanceRate`). This is just good practice, and in JavaScript it can have [a weird effect or two](https://spin.atomicobject.com/2011/04/10/javascript-don-t-reassign-your-function-arguments/).

Reply:
Style issues fixed.